core/src/main/java/jenkins/model/ParameterizedJobMixIn.java

```diff
@@ -218,6 +218,7 @@ public final void doBuild(StaplerRequest req, StaplerResponse rsp, @QueryParamet
 
         Queue.Item item = Jenkins.get().getQueue().schedule2(asJob(), delay.getTimeInSeconds(), getBuildCause(asJob(), req)).getItem();
         if (item != null) {
+            // TODO JENKINS-66105 use SC_SEE_OTHER if !ScheduleResult.created
```

Comment:
Forgotten in #5615 or just “out of scope”? If such a change is desired, I would leave it to a separate PR for clarity.